client/src/main/scala/drt/client/components/TerminalContentComponent.scala

```diff
@@ -44,18 +44,18 @@ object TerminalContentComponent {
         cs => cs.crunchMinutes.toSeq.map(_.hashCode())
       ).toList.mkString("|")
 
-      val flightsHash: Option[List[(Int, String, String, String, String, String, String, String, String, Long, Int)]] = crunchStatePot.toOption.map(_.flights.toList.map(f => {
+      val flightsHash: Option[List[(Int, String, String, String, Long, Long, Long, Long, Long, Long, Int)]] = crunchStatePot.toOption.map(_.flights.toList.map(f => {
         (f.splits.hashCode,
           f.apiFlight.Status,
-          f.apiFlight.Gate,
-          f.apiFlight.Stand,
-          f.apiFlight.SchDT,
-          f.apiFlight.EstDT,
-          f.apiFlight.ActDT,
-          f.apiFlight.EstChoxDT,
-          f.apiFlight.ActChoxDT,
-          f.apiFlight.PcpTime,
-          f.apiFlight.ActPax
+          f.apiFlight.Gate.getOrElse(""),
+          f.apiFlight.Stand.getOrElse(""),
+          f.apiFlight.Scheduled,
+          f.apiFlight.Estimated.getOrElse(0L),
```

Comment:
I *think* you can actually just leave these as options for the purpose of this function which is just to create a unique hash of all the data in the component. @richbirch might know better than me.

Reply:
I realise it doesn't matter programatically in this context, however, I think it's just misleading to do getOrElse(0) on a timestamp option, because the 0 case clearly would never be correct in any circumstances and so is not a useful fall back.